src/style.css

```diff
@@ -0,0 +1,109 @@
+@import "../src/variables/variables.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", Helvetica, Arial, sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid var(--white--color);
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card_apple {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card_descript {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--textCol--color);
+}
+
+.card_code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+
+  color: var(--grey--color);
+  margin-bottom: 16px;
+}
+
+.card_rew {
+  margin-bottom: 24px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.review {
+  color: var(--textCol--color);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card_price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--grey--color);
+}
+
+.price_value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--textCol--color);
+}
+
+.card_buy {
+  width: 100%;
+  padding: 12px 0;
+
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  text-decoration: none;
+  color: var(--white--color);
+  background-color: var(--blue--color);
+
+  display: inline-block;
```

Comment:
remove this aswell, the default value of `block` should be fine, you also won't need to give the width 100% because of this

Reply:
It doesn't work without that!
https://prnt.sc/pfQLjjVPeBMU